compel/test/stack/.gitignore

```diff
@@ -0,0 +1,4 @@
+parasite.h
+parasite.po
```

Comment:
Do we also need parasite.o here?

Reply:
There's `*.o` in the root .gitignore that filters it out; these test specific .gitignore files look like they only mention additional things to ignore.